main.tf

```
@@ -501,6 +504,45 @@ resource "aws_lb_listener_rule" "this" {
   listener_arn = try(aws_lb_listener.this[each.value.listener_key].arn, each.value.listener_arn)
   priority     = each.value.priority
 
+  dynamic "transform" {
+    for_each = each.value.transforms != null ? each.value.transforms : {}
```

I know we've been inconsistent elsewhere but lets match the API with the singular form 
```suggestion
    for_each = each.value.transform != null ? each.value.transform : {}
```